src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -191,6 +191,40 @@
                       (read-record add-node-expr-fn)
                       d/cache-node)))}}))
 
+(defn- create-identity-based-handler
+  [clazz
+   tag
+   read-fn
```

Comment:
I think whatever we choose, we should be consistent across function signatures to avoid confusion.

Write before read sort of makes sense since writing does have to happen before read - technically.

Reply:
reordered: 
https://github.com/EthanEChristian/clara-rules/commit/bc8ba8fe933c6fd25d5911b6e80401f2563f8112#diff-97779c182cfdc7f57412ed5c7fe3ff07L197